### DISCUSSION TITLE
Convert existing SQLAlchemy ORM to SQLModel ORM

### DISCUSSION THREAD

**Discussion by havok2063** [fastapi/sqlmodel#1942] [Questions]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the SQLModel documentation, with the integrated search.
- [X] I already searched in Google "How to X in SQLModel" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to SQLModel but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to SQLModel but to [SQLAlchemy](https://github.com/sqlalchemy/sqlalchemy).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
# example potential pseudo-code of desired functionality

# import existing SQLAlchemy ORM Table
from project.sqlalchemy.models import Table

# convert SQLA Table ORM to SQLModel Table
SQLM_Table = SQLModel.convert(Table)

# or batch convert a set of SQLAlchemy ORMs
from project.sqlalchemy import models
SQLM_models = sqlmodel_batch_convert(models)
```

### Description

In your docs, you only describe creating a db, tables and `SQLModel` ORM from scratch.  If you already have a set of defined SQLAlchemy ORMs for an existing db, is there a mechanism for converting existing `SQLAlchemy` ORMs into `SQLModel` ORMs?  I looked through the docs but couldn't find anything related to that.    I provide some desired functionality as pseudocode, but maybe this functionality already exists?  

### Operating System

macOS

### SQLModel Version

latest

### Python Version

3.8

**Comment by supermacro** [comment]
I am currently trying to implement `SQLModel.convert` ... if I end up developing something useful I will share here.

Also curious to hear if in 2025 there's perhaps a better approach for integrating existing sqlalchemy models with sqlmodel
